### PR TITLE
Document name option for edl21

### PR DESCRIPTION
**Pull request by mtdcr**
## Proposed change

In order to distinguish multiple smart meters more easily, a name parameter was added as part of a PR which fixes support for multiple smart meters in general.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/33594

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
Please rebase your branch to the `next` branch

**Comment by mtdcr** [comment]
@klaasnicolaas: Done, thank you!

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_integrations/edl21.markdown [hunk 1]
```diff
@@ -29,10 +29,15 @@ To set it up, add the following information to your `configuration.yaml` file:
 ```yaml
 sensor:
   - platform: edl21
+    name: "Heat pump"
```
Comment (klaasnicolaas) [inline source/_integrations/edl21.markdown]: In the first example we only want required config variables

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/edl21.markdown (hunk 1 above)
Comment (mtdcr) [reply source/_integrations/edl21.markdown]: I removed it.

**Review by klaasnicolaas — DISMISSED** [review]

**Review by fabaff — APPROVED** [review]

**Pull request merged**